plugins/changed_managed_files/changed_managed_files_renderer.rb

```diff
@@ -28,11 +28,11 @@ def content(description)
     list do
       file_status = description["changed_managed_files"].extracted
       if !file_status.nil?
-        puts "Files extracted: #{file_status ? "yes" : "no"}"
-      end
-
-      if files && files.empty?
-        puts "There are no changed managed files."
+        if files.empty?
```

Comment:
I think we should also show that message when the `file_status` is nil. It's probably a theoretical case but still... ;)